troposphere/static/js/components/modals/nullProject/NullProjectMigrateResourceModal.react.js

```diff
@@ -33,16 +34,10 @@ define(
       getInitialState: function () {
         var initialState = {
           projectName: "",
-          projects: stores.ProjectStore.getAll(),
-          projectId: null
+          projects: null,
+          projectId: -999
```

Comment:
Any particular reason why -999 is chosen over null? I can see people being confused by a magic value rather than an uninitialized value.